site/content/posts/contributor-of-the-month.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Introducing the Contributor of the Month initiative"
+slug: contributor-of-the-month
+date: 2022-03-01
+author: Cartographer Team
+authorLocation: https://github.com/vmware-tanzu/cartographer/blob/main/MAINTAINERS.md
+image: /img/posts/contributor-of-the-month/Contributor-of-the-month-v3.png
+excerpt: "A monthly program to reward contributions"
+tags: ['community']
+---
+
+Since October 2021, when the Cartographer project was made public, we have been very fortunate to count with individuals who have helped improve the codebase and docs while also telling the Cartographer story in conferences, live streams and other forms of content. All of them have done this out of intrinsic motivation, without expecting any reward or public recognition and we are forever grateful to them for this.
```

Comment:
```suggestion
Since October 2021, when the Cartographer project was made public, we have been very fortunate to count on individual contributors who have helped improve the codebase and docs while also telling the Cartographer story at conferences, in live streams and other forms of content. All of them have done this out of an intrinsic motivation, without expecting any reward or public recognition. We are forever grateful to them for this.
```